modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java

```diff
@@ -378,6 +380,15 @@ public void postProcessSearchQuery(
 				else {
 					searchQuery.add(booleanQuery, BooleanClauseOccur.SHOULD);
 				}
+
+				if (keywords.startsWith(StringPool.QUOTE) &&
+					keywords.endsWith(StringPool.QUOTE) &&
+					(keywords.length() > 1)) {
```

Comment:
I can't find examples of this pattern in the code base, are you sure this is the right way to make an exact search?

Reply:
@andrea-ale-sbarra yeah this is not a existing pattern. Hmm, if this is too hacky we might be able to do something with additional-type-mappings.

Reply:
I think we need to find a better solution